composer.json

```
@@ -0,0 +1,67 @@
+{
+    "name": "openeuropa/oe_starter_content",
+    "description": "OpenEuropa Drupal module template.",
+    "type": "drupal-module",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "drupal/core": "^9.2",
+        "php": ">=7.4"
+    },
+    "require-dev": {
+        "composer/installers": "^2.0",
+        "drupal/config_devel": "^1.8",
+        "drupal/core-composer-scaffold": "^9.2",
+        "drupal/core-dev": "^9.2",
+        "drupal/drupal-extension": "^4.1",
+        "drush/drush": "^10.3",
+        "openeuropa/code-review": "^2.0",
+        "openeuropa/task-runner-drupal-project-symlink": "dev-EWPP-1884",
+        "phpspec/prophecy-phpunit": "^1 || ^2"
+    },
+    "scripts": {
+        "post-install-cmd": "./vendor/bin/run drupal:site-setup",
+        "post-update-cmd": "./vendor/bin/run drupal:site-setup"
+    },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Drupal\\oe_starter_content\\": "./src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Drupal\\Tests\\oe_starter_content\\": "./tests/"
```

We need to force tests under src
```suggestion
            "Drupal\\Tests\\oe_starter_content\\": "./tests/src"
```